spec/models/miq_alert_spec.rb

```diff
@@ -159,7 +159,7 @@
       it "miq_alert_status.description = miq_alert.description event if overriden by ems_event.description" do
         @alert.evaluate(
           [@vm.class.base_class.name, @vm.id],
-          :ems_event => FactoryGirl.create(:ems_event, :message => "oh no!", :type => 'WhateverEvent')
+          :ems_event => FactoryGirl.create(:ems_event, :message => "oh no!")
```

Comment:
I think the rationale was that we should never create an event that's of the base type because the table uses STI

Reply:
EmsEvent appears to be the leaf subclass.
```ruby
EmsEvent.descendants
=> []
EventStream.descendants.collect(&:name)
=> ["EmsEvent", "MiqEvent", "RequestEvent", "CustomEvent"]
```